tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -144,24 +144,42 @@ def test__get_collections_with_auth__OK(self):
         self.assertEqual(200, res_auth.status_code)
         self.assertEqual(6, len(res_auth.json["collections"]))
         with self.subTest("The 'revising_in' attribute is None for unauthorized public collections"):
+            conditions_tested = 0
```

Comment:
rather than doing this condition thing, make each condition a subtest.

Reply:
Also this test is too long, please break it up into small tests.

Reply:
>rather than doing this condition thing, make each condition a subtest.

That would be a lot of code duplication. Also I'm not seeing how we would do this without the `conditions_tested` flag. You need that to ensure that you actually run the `assert` that is doing the testing since it is nested, necessariyl, into a conditional.

Reply:
I see why you're using `conditions_tested`, but it's still an odd pattern. I also don't like seeing the test iterating through the whole list every time, but this is not intended to be performant code so I'll let that go.  Breaking it up would not be much code duplication, but I don't see a compelling reason to do that at this time.

Carry on